akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ClientSpec.scala

```diff
@@ -192,6 +192,170 @@ class Http2ClientSpec extends AkkaSpecWithMaterializer("""
         )
       }
 
+      "accept response with one HEADERS and one CONTINUATION frame" in new TestSetup with NetProbes {
+        user.emitRequest(0x1, Get("https://www.example.com/"))
+        network.expect[HeadersFrame]()
+
+        val headerBlock = HPackSpecExamples.C61FirstResponseWithHuffman
+        val fragment1 = headerBlock.take(8) // must be grouped by octets
+        val fragment2 = headerBlock.drop(8)
+        network.sendHEADERS(0x1, endStream = true, endHeaders = false, fragment1)
```

Comment:
I couldn't figure out why end-stream should not lead to an error when you send CONTINUATION from the spec, did I miss something?

Reply:
CONTINUATION does not define an END_STREAM flag so it needs to go to corresponding HEADERS flag. A few things seem to be botched around the CONTINUATION spec bits so do not lose too much sleep about that...

Reply:
Ah, thanks for the clarification.